tmt/steps/discover/__init__.py

```diff
@@ -192,7 +231,33 @@ def base_command(cls, method_class=None, usage=None):
         @click.option(
             '-h', '--how', metavar='METHOD',
             help='Use specified method to discover tests.')
+        @click.option(
+            '--fmf-id', default=False, is_flag=True,
```

Comment:
I think this should be just in `DiscoverFmf`. IMO there is no point to have common option for all discover plugins when it can be used only with `how:fmf`